ale_linters/verilog/xvlog.vim

```diff
@@ -2,7 +2,12 @@
 " Description: Adds support for Xilinx Vivado `xvlog` Verilog compiler/checker
 
 call ale#Set('verilog_xvlog_executable', 'xvlog')
-call ale#Set('verilog_xvlog_options', '')
+
+if &filetype == 'systemverilog'
+    call ale#Set('verilog_xvlog_options', '-sv')
+else
+    call ale#Set('verilog_xvlog_options', '')
+endif
```

Comment:
The `-sv` option should be added dynamically in the `GetCommand` function based on the the filetype for `a:buffer`. I'll close this PR and you can open another in future with this and a test for the change.